integration_tests/support/events.ts

```diff
@@ -1,13 +1,18 @@
-import AWS from 'aws-sdk'
-import { SendMessageRequest } from 'aws-sdk/clients/sqs'
+import * as AWS_SQS from '@aws-sdk/client-sqs'
+import config from '../../server/config'
 
-const sqs = new AWS.SQS({
+const { SQS } = AWS_SQS
+
+const sqs = new SQS({
   region: 'eu-west-2',
-  accessKeyId: 'foo',
-  secretAccessKey: 'bar',
+  credentials: {
+    accessKeyId: 'foo',
+    secretAccessKey: 'bar',
+  },
```

Comment:
this credentials shouldn't be set as its picked up from features.env

Reply:
No, It's not picking up credentials from feature.env file.

Reply:
my bad! this runs from within cypress, feature.env is only available to the running application!